broker/api.py

```diff
@@ -24,9 +24,26 @@
 
 
 from broker import validators
+from broker.aws import cloudfront
 from broker.extensions import config, db
-from broker.lib.cdn import is_cdn_instance, is_cdn_dedicated_waf_instance
+from broker.lib.alb import (
+    validate_migration_to_alb_params,
+    update_alb_params_for_migration,
+)
+from broker.lib.cache_policy_manager import CachePolicyManager
```

Comment:
pulling `cloudfront` and `CachePolicyManager` in here feels like we're mixing abstraction levels with the rest of the `api`
this is very much opinion, and I'm having a hard time explaining why it feels like the wrong level or how to fix it, so feel free to ignore

Reply:
I agree. This case is somewhat special because we need to talk to AWS to get the policy IDs so that we can set the ID for the specified policy on the instance around here:

https://github.com/cloud-gov/external-domain-broker/blob/6960f98d2286aaa9d3c31040a71bdb8775918d48/broker/lib/cdn.py#L94

And that needs to happen in the context of the API request.  Still thinking of how to change it

I could do this in `broker/extensions`

```
cache_policy_manager = CachePolicyManager(cloudfront)
```

That's just moving things around though

Reply:
I think that move might be worthwhile still, because then we're keeping `cloudfront` itself out of the `api` scope